controllers/util/mw_util.go

```diff
@@ -515,7 +518,7 @@ func (mwu *MWUtil) GetVRGManifestWorkCount(drClusters []string) int {
 	return count
 }
 
-func (mwu *MWUtil) DeleteManifestWorksForCluster(clusterName string) error {
+func (mwu *MWUtil) DeleteVRGManifestWorksForCluster(clusterName string) error {
```

Comment:
The original intent of the function was to delete multiple ManifestWork resources, given that we would conditionally delete VRG MW and NS MW separately we could instead use `DeleteManifestWork` directly than adding functions for the same (no other manifest has such a delete function, hence tending towards this option than defining a delete per resource type).

Or, rename the Works part of the function to just Work, as we are deleting a singleton

Reply:
With the updated changes I'm directly calling the `DeleteManifestWork` function, removed other layers of wrapper functions. Please see if it looks good